test/scripts/openshift-ci/deploy.odh.sh

```
@@ -22,100 +22,63 @@ set -eu
+  echo "Patching operator CSV to mount custom manifests volume..."
+  CSV=$(oc get subscription "${OPERATOR_NAME}" -n "${OPERATOR_NAMESPACE}" -o jsonpath='{.status.installedCSV}')
+  echo "Found CSV: $CSV"
 
-# Check if volume is already mounted
-if oc get csv "$CSV" -n ${ODH_OPERATOR_NAMESPACE} -o json | jq -e '.spec.install.spec.deployments[0].spec.template.spec.volumes[] | select(.name=="kserve-custom-manifests")' > /dev/null 2>&1; then
-  echo "Volume already mounted, skipping patch"
-else
-  echo "Applying CSV patch to mount custom manifests volume..."
-  oc patch csv "$CSV" -n ${ODH_OPERATOR_NAMESPACE} --type json --patch-file "${SCRIPT_DIR}/odh-operator-custom-manifests/csv-patch.json"
-fi
+  if oc get csv "$CSV" -n ${OPERATOR_NAMESPACE} -o json | jq -e '.spec.install.spec.deployments[0].spec.template.spec.volumes[] | select(.name=="kserve-custom-manifests")' > /dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)